build/ci/check_demo_pr_checks.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_checks_passed=true
```

Comment:
I would start with `false`, just to be on the safe side

Reply:
done